### PR TITLE
Restructure Discord.markdown to clarify process

### PR DESCRIPTION
**Pull request by stevietv**
## Proposed change

Update the docs to make the process more clear.

I've reworked this document into three steps.
1. Create the Application and Bot
2. Add the Bot to a server
3. Add the integration and link to the bot.

The docs also referred to client id which is now called Application ID in the discord developer portal, this is updated.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes [#68547](https://github.com/home-assistant/core/issues/68547)

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @stevietv, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#22410).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/discord.markdown [hunk 1]
```diff
@@ -16,31 +16,33 @@ ha_integration_type: integration
...
+Once the application is ready, create a [bot](https://discordapp.com/developers/docs/topics/oauth2#bots) user (**Create a Bot User**).
 
-<div class='note'>
+Retrieve the **Application ID** from the 'General Information' section and the (hidden) **Token** of your bot for later.
 
+<div class='note'>
+  
 The name you give your application on the [Discord My Apps page](https://discordapp.com/developers/applications/me) will determine the name of the notify service. For example: if you enter "Discord Chat", the service will be named `notify.discord_chat`.
-
+
```
Comment (frenck) [inline source/_integrations/discord.markdown]:
```suggestion

```

File: source/_integrations/discord.markdown [hunk 2]
```diff
@@ -16,31 +16,33 @@ ha_integration_type: integration
 
 The [Discord service](https://discordapp.com/) is a platform for the notify component. This allows integrations to send messages to the user using Discord.
 
-In order to get a token you need to go to the [Discord My Apps page](https://discordapp.com/developers/applications/me) and create a new application. Once the application is ready, create a [bot](https://discordapp.com/developers/docs/topics/oauth2#bots) user (**Create a Bot User**).
+## Creating the Discord Application
```
Comment (frenck) [inline source/_integrations/discord.markdown]:
```suggestion
## Prerequisites
```

This change looks a bit weird considering the next paragraph. Maybe these should be 2 sub paragraphs under prerequisites?

**Comment by stevietv** [comment]
any news on a review and commit?

**Review by frenck — COMMENTED** [review]
Two small tweaks

**Comment by stevietv** [comment]
> Two small tweaks

thanks for the review, suggestions adopted :)

**Comment by frenck** [comment]
The latter had a question, did you see that?

**Comment by stevietv** [comment]
> The latter had a question, did you see that?

Oh I see it now. I've thrown creating an application and setting up the bot into the pre-requisites heading

**Review by frenck — APPROVED** [review]
Thanks, @stevietv 👍 

../Frenck

**Pull request merged**